### ISSUE TITLE
Treat @JsonRootName like @Label

### ISSUE THREAD

**Issue by vguna** [stoicflame/enunciate#221]
It would be very nice, if enunciate would offer an option to support the native jackson @JsonRootName annotation. Basically it is the same as enunciate's @Label.

Currently I have to do this:

```java
@JsonRootName("account")
@Label("account")
public class AccountDto {
...
```

I know that @Label is more powerful since it can be interpreted for XML as well as JSON, but some native support for the only-json guys would be great :D.

**Comment by stoicflame** [comment]
Fixed at 09dd986.

**Comment by stoicflame** [comment]
[Enunciate 2.2 is now available](https://github.com/stoicflame/enunciate/releases/tag/v2.2.0).
